### PR TITLE
Dialogue connections need to be easier to follow

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

File: go/base/static/css/conversations.less [hunk 1]
```diff
@@ -6,8 +6,32 @@
...
+      opacity: 0.5;
+    }
+
+    ._jsPlumb_hover {
+      z-index: 3;
+      opacity: 0.75;
+    }
+
+    *:not(._jsPlumb_connector) {
+      z-index: 2;
```
Comment (hodgestar) [inline go/base/static/css/conversations.less]: Will this not override some of the z-indexes above?

File: go/base/static/js/src/campaign/dialogue/style.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// go.campaign.dialogue.states.dummy
```
Comment (hodgestar) [inline go/base/static/js/src/campaign/dialogue/style.js]: Should be `go.campaign.dialogue.style`?

**Comment by hodgestar** [comment]
:+1: other than the `z-index` question and the fix to the styles file header.

File: go/base/static/css/conversations.less (hunk 1 above)
Comment (justinvdm) [inline go/base/static/css/conversations.less]: Good point, will change this. I don't like using `:not`, but it is the only workaround to get the z indices to work properly.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**